pom.xml

```diff
@@ -47,7 +47,7 @@
 
         <deps.apache-sshd.version>0.12.0</deps.apache-sshd.version>
         <deps.jsch.version>0.1.52</deps.jsch.version>
-        <deps.ssh-credentials-plugin.version>1.13-SNAPSHOT</deps.ssh-credentials-plugin.version>
+        <deps.ssh-credentials-plugin.version>2.0-SNAPSHOT</deps.ssh-credentials-plugin.version>
```

Comment:
Because property substitution is quazai undefined. Does a system property affect the transitive dependency resolution of a dependent pom? What happens if I locally define a property in my Pom with the same name and a different value?

The uncertainty is worth the risk where two or more versions need to be synchronised but if you only have the version in one and only one place it is making the Pom more verbose than it needs to be... Which I an anti-pattern